config/test.json

```diff
@@ -1,5 +0,0 @@
-{
-  "security": {
-    "secret": "test"
-  }
-}
```

Comment:
Ha-ha, it wasn't removed, it just now picked up by the https://github.com/Avaq/node-server-skeleton/pull/16/files#diff-a084b794bc0759e7a6b77810e01874f2L29, i.e. now I understand why you had two `.gitignores`